src/screens/CreatePost/elements/PollItem.js

```diff
@@ -14,13 +14,13 @@ export default function PollItem({
   onremovepoll = () => {},
   onpollchanged = () => {},
   showdeleteicon,
-  showcharactercount = false
+  showcharactercount = false,
+  isAnonym
 }) {
   const [isTextInputFocus, setIsTextInputFocus] = React.useState(false);
 
   return (
-    // TODO: Garry warna focus sesuai mode
-    <View style={isTextInputFocus ? S.focuspollitemcontainer : S.pollitemcontainer}>
+    <View style={isTextInputFocus ? S.focuspollitemcontainer(isAnonym) : S.pollitemcontainer}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isAnonym` prop is used to conditionally style the `View` component. However, it's not clear from this code snippet what the `isAnonym` prop represents. It would be beneficial to add a comment explaining its purpose for future maintainability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->